lib/worker.go

```diff
@@ -94,12 +100,18 @@ func (w *LocalWorker) Start(coord WorkCoordinator) error {
 					log.Error(err.Error())
 					continue
 				}
-				if err := w.queue.Send(tg); err != nil {
+				if err := w.queues[i].Send(tg); err != nil {
 					log.Error(err.Error())
 					continue
 				}
+				i++
+				if i == len(w.queues) {
+					i = 0
```

Comment:
Could just be `i = (i + 1) % len(w.queues)`

Reply:
wow that's gorgeous